app/controllers/conference_registrations_controller.rb

```diff
@@ -10,7 +10,7 @@ def new
       redirect_to edit_conference_conference_registrations_path(@conference.short_title)
       return
     # ichain does not allow us to create users during registration
-    elsif CONFIG['authentication']['ichain']['enabled'] && !current_user
+    elsif (ENV['OSEM_ICHAIN_ENABLED'] == 'true') && !current_user
```

Comment:
No it's always a string. Usually this is done by existance. Either it is set, no matter the content, or not. Would you like that more?